test/Test/Store/Substitutor.hs

```diff
@@ -141,10 +141,10 @@ spec = do
             expr =
               MyRecord mempty $
                 M.fromList
-                  [ (mkName "first", MyApp mempty (MyVar mempty (mkName "id")) (int 1)),
-                    (mkName "second", MyApp mempty (MyVar mempty (mkName "id")) (int 2))
+                  [ ("first", MyApp mempty (MyVar mempty "id") (int 1)),
```

Comment:
OH YEAH. Made `IsString` instances for our main identifiers to make using them less arduous.